### ISSUE TITLE
Could not retrieve memory stats

### ISSUE THREAD

**Issue by pipeosorio1** [ejhayes/bull-monitor#247]
when using the nestjs/bull library it is not possible to monitor the jobs

<img width="1288" alt="Captura de Pantalla 2023-05-10 a la(s) 12 03 51 p m" src="https://github.com/ejhayes/bull-monitor/assets/88846070/17d190e1-ad6a-4b38-83a6-14c0a0d8033e">

**Comment by OlexanderKulyk** [comment]
same error :-(

**Comment by ejhayes** [comment]
Related to #158. This project currently only supports bullmq so anything using a bull queue either won't work or will have strange behavior.

**Comment by pipeosorio1** [comment]
According to what I have reviewed, the problem is that the bull does not have the "meta" tag in redis, if it is created manually in redis, it is capable of displaying the graphics on the monitor.

<img width="523" alt="image" src="https://github.com/ejhayes/bull-monitor/assets/88846070/8fc3471e-b6fc-4bb5-8fb8-2ce7d8862c09">

With this Provider the registration in the job


```javascript
import { Queue } from 'bull';

export class QueueProvider {
  constructor(protected readonly queue: Queue) {
    this.initializeMetaHash();
  }

  protected async initializeMetaHash(): Promise<void> {
    const prefix = process.env.APP_NAME;
    const metaKey = `${prefix}:${this.queue.name}:meta`;
    const exists = await this.queue.client.exists(metaKey);

    // Si el hash "meta" no existe se crea
    if (!exists) {
      const metaValue = { 'opts.maxLenEvents': 10000 };
      await this.queue.client.hmset(metaKey, metaValue);
    }
  }
}
```

**Comment by ejhayes** [comment]
Yep, meta key was introduced by bullmq. As mentioned earlier implementing #158 is the best path forward
